### DISCUSSION TITLE
Problem with enabling lsp-saga

### DISCUSSION THREAD

**Discussion by boozedog** [nix-community/nixvim#522] [Q&A] [answered]
I've got this:
```
lspsaga.enable = true;
```

Getting this when I start nvim:
```
Error detected while processing /home/david/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/david/.config/nvim/init.lua:46: attempt to call fie
ld 'init_lsp_saga' (a nil value)
stack traceback:
        /home/david/.config/nvim/init.lua:46: in main chunk
Press ENTER or type command to continue
```

Any ideas?

**Comment by GaetanLepage** [comment]
Hello,
Indeed, this is due to a change of the plugin underlying `plugins.lspsaga` (and actually `vimPlugins.lspsaga-nvim`).
See [here](https://github.com/NixOS/nixpkgs/pull/247916).

What you need to do is update your flake's inputs (both nixvim and nixpkgs) to get the right match for the lspsaga support in nixvim.

**Reply by GaetanLepage** [reply]
When you will rebuild your configuration, your `init.lua` will be correctly generated so as to accommodate the new plugin's options.

**Reply by boozedog** [reply]
Thanks @GaetanLepage ! Here's what I have in my inputs, is this wrong?

```nix
inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
    nixvim.url = "github:nix-community/nixvim";
...
```

**Reply by GaetanLepage** [reply, marked as answer]
No, your inputs are fine. You simply need to run `nix flake update` to fetch their latest version.

**Reply by GaetanLepage** [reply]
Is it working now @boozedog ?
